flutter/realm_flutter/example/README.md

```diff
@@ -18,3 +18,34 @@ A few resources to get you started if this is your first Flutter project:
 For help getting started with Flutter, view our
 [online documentation](https://flutter.dev/docs), which offers tutorials,
 samples, guidance on mobile development, and a full API reference.
+
+
+## Setup 
+Run these commands to setup the application
```

Comment:
All of this is not needed. This is an example in the repo which references the package from the repo. Also any steps of `pub get` are not needed since `flutter run` will do it as well

Reply:
I was about to approve but I suggest we remove these changes here

Reply:
Ok. I removed it.